src/components/Facility/ConsultationDetails/PrintQuestionnaireQuestionnaireResponses.tsx

```diff
@@ -186,10 +186,7 @@ export function EncounterDetails({
         />
         <DetailRow
           label={t("mobile_number")}
-          value={
-            encounter?.patient.phone_number &&
-            formatPhoneNumber(patient!.phone_number)
-          }
+          value={encounter && formatPhoneNumber(patient.phone_number)}
```

Comment:
shouldn't this be `{patient && formatPhoneNumber(patient.phone_number)}`

Reply:
as per the types, `patient` is always present. so we don't need to do a check even before showing the value. updated.